### PR TITLE
[FEATURE] Add "GitHub" button in the Settings screen (Issue #2675)

### LINKED ISSUE #2675: [FEATURE] Add "GitHub" button in the Settings screen
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues

### What do you want to be added or improved?

As a user, I want to have a "GitHub" button in Ivy Wallet's Settings screen that open the Ivy Wallet GitHub repo.

### Why do you need it?

- I want to see the project's open-source repo.
- The more people that see the repo, the more contributors we'll get.

### How do you imagine it?

I colorful "GitHub' button in the Settings screen right below "Share Ivy Wallet" in the "Other" section.
When click the button should open:

https://github.com/Ivy-Apps/ivy-wallet

### PR DESCRIPTION
**Pull request by TharunBalaji2004**
## Pull Request (PR) Checklist

- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md).
- [x] I've read the [Architecture Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Architecture.md).
- [x] My code builds and is tested on a real Android device.
- [x] I confirm that I've run the code locally and everything works as expected.

## What's changed?

| Others Section (Before) | Others Section (After) |
| :-----: | :-----: |
| Screenshot from 2023-09-27 15-46-07 | GitHub Button Feature |

Describe with a few bullets **what's new:**
- Added new GitHub card under Others section, which on clicking redirects to IvyWallet repo page
- It supports both Light Mode and Black Mode

## Risk Factors

**What may go wrong if we merge your PR?**

- Everything works fine ✅ tested in both virtual and physical device

**In what cases your code won't work?**

- Most probably not

## Does this PR closes any GitHub Issues?

- Closes  #2675

### CODE REVIEW

File: .idea/deploymentTargetDropDown.xml
Comment (ILIYANGERMANOV) [inline .idea/deploymentTargetDropDown.xml:1]: Can you remove this file from the PR?

**Review by ILIYANGERMANOV — APPROVED** [review]
LGTM! Nice work 💯 Just remove the committed idea file. You can add it to `.gitignore`, too

**Comment by TharunBalaji2004** [comment]
@ILIYANGERMANOV Oops I didn't noticed that, I have deleted that file from PR. Next time I'll mind that

**Review by ILIYANGERMANOV — APPROVED** [review]
Excellent work 👏

**Comment by ILIYANGERMANOV** [comment]
Merging after the CI pass ✅

**Comment by TharunBalaji2004** [comment]
Thank you 😊 I'm glad to say that I have successfully did my first open-source contribution

**Pull request merged**